lib/docusign_transaction_rooms/paginated_resource.rb

```diff
@@ -0,0 +1,72 @@
+module DocusignTransactionRooms
+  class PaginatedResource
+    include Enumerable
+
+    COUNT = 100
```

Comment:
is this per_page?

Reply:
yep, docusign uses count for some reason. Just matching their API

Reply:
Can you just put a comment to the side to confirm it's per_page.